runtime/protocol/http-client-engines/test-suite/jvm/src/aws/smithy/kotlin/runtime/http/test/suite/Downloads.kt

```diff
@@ -53,6 +53,7 @@ internal fun Application.downloadTests() {
                 call.response.header("x-bar", "bar")
             }
 
+            // FIXME This is the only thing keeping this directory in `jvm`, could be `jvmAndNative`
             get("/gzipped") {
                 val uncompressed = ByteArray(1024) { it.toByte() }
                 val compressed = ByteArrayOutputStream().use { baStream ->
```

Comment:
**Question**: Don't we already know we need KMP GZip support and already have `expect`/`actual` declarations for it? Can we use those here and then move this to the **jvmAndNative** source set? Or do other blockers remain?

Reply:
Maybe we can use our `expect`/`actual` declarations, I'll take a look. Right now this is using `java.util.zip.GZIPOutputStream` which is definitely JVM-only

Reply:
Moved to `jvmAndNative` by using our existing `GzipSdkSource` class